src/openrct2/Game.cpp

```diff
@@ -622,7 +622,11 @@ static void GameLoadOrQuitNoSavePromptCallback(int32_t result, const utf8* path)
 
 static void NewGameWindowCallback(const utf8* path)
 {
-    WindowCloseByClass(WindowClass::EditorObjectSelection);
+    // Closing this will cause a Ride window to pop up, so we have to do this to ensure that
```

Comment:
Closing what? And it does sound quite odd, do you know why a new window shows up?

Reply:
Assuming _something_ has been built, closing the construction window opens the associated ride window.

Reply:
I would have thought that was obvious, considering the comment is right above a call to close windows of the RideConstruction class.